packages/RichText-Tests.package/RTEHeaderComponentTest.class/instance/testH1LargerH2LargerH3.st

```diff
@@ -1,18 +1,15 @@
 testing
 testH1LargerH2LargerH3
```

Comment:
maybe rename to `testHeaderFontsizeDifferences` or something similar? H1, H2, H3 seems to be a bit to explicit